openpype/hosts/maya/hooks/pre_copy_mel.py

```diff
@@ -10,10 +10,10 @@ class PreCopyMel(PreLaunchHook):
     app_groups = ["maya"]
 
     def execute(self):
-        project_name = self.launch_context.env.get("AVALON_PROJECT")
+        project_doc = self.data["project_doc"]
         workdir = self.launch_context.env.get("AVALON_WORKDIR")
         if not workdir:
             self.log.warning("BUG: Workdir is not filled.")
             return
 
-        create_workspace_mel(workdir, project_name)
+        create_workspace_mel(workdir, project_doc["name"])
```

Comment:
We could likely optimize this call further by allowing `create_workspace_mel` to just take project settings instead? Likely that function is still querying the the settings?

Reply:
Done